src/test/scala/mesosphere/marathon/api/validation/AppUpdateValidatorTest.scala

```diff
@@ -1,133 +0,0 @@
-package mesosphere.marathon
-package api.validation
-
-import com.wix.accord.validate
-import mesosphere.UnitTest
-import mesosphere.marathon.api.v2.AppNormalization
-import mesosphere.marathon.api.v2.validation.AppValidation
-import mesosphere.marathon.core.plugin.PluginManager
-import mesosphere.marathon.raml.{App, AppCContainer, AppUpdate, ContainerPortMapping, EngineType, Raml}
-import mesosphere.marathon.state.AppDefinition
-import org.scalatest.Matchers
-import play.api.libs.json.Json
-
-class AppUpdateValidatorTest extends UnitTest with Matchers {
-  implicit val appUpdateValidator = AppValidation.validateCanonicalAppUpdateAPI(Set.empty, () => AppNormalization.Configuration(None, "mesos-bridge-name").defaultNetworkName, Map.empty)
-  implicit val validAppDefinition = AppDefinition.validAppDefinition(Set.empty)(PluginManager.None)
-
-  "validation that considers container types" should {
```

Comment:
What about these tests (and the ones above)? Are *all* those cases properly covered in other tests?

Reply:
I checked every test to make sure they don't disappear :)